packages/react/src/Bento/Bento.tsx

```diff
@@ -0,0 +1,298 @@
+          className: styles['Bento__Content-icon'],
+          size: LeadingVisual['size'] || 44,
+        })}
+      {React.isValidElement(HeadingChild) && React.cloneElement(HeadingChild as React.ReactElement<BentoHeadingProps>)}
+      {React.isValidElement(TextChild) &&
+        React.cloneElement(TextChild as React.ReactElement<TextProps>, {
+          variant: TextChild.props.variant || 'muted',
+          as: TextChild.props.as || 'p',
+          size: TextChild.props.size || '300',
+          className: clsx(styles['Bento_Content-text'], TextChild.props.className),
+        })}
+      {React.isValidElement(LinkChild) &&
+        React.cloneElement(LinkChild as React.ReactElement<LinkProps>, {
+          variant: LinkChild.props.variant || 'accent',
+          className: clsx(
+            styles['Bento__call-to-action'],
+            fixedBottomLink && styles['Bento__call-to-action--fixed'],
+            LinkChild.props.className,
+          ),
+        })}
+    </div>
+  )
+}
+
+type BentoHeadingProps = BaseProps<HTMLHeadingElement> & HeadingProps
+
+const defaultHeadingTag = 'h3'
+const defaultHeadingSize = '5'
+const defaultHeadingWeight = 'medium'
+
+const _Heading = forwardRef(
+  (
+    {
+      as = defaultHeadingTag,
+      size = defaultHeadingSize,
+      weight = defaultHeadingWeight,
+      className,
+      children,
+      ...props
+    }: PropsWithChildren<BentoHeadingProps>,
+    ref: Ref<HTMLHeadingElement>,
+  ) => {
+    const childrenArray = useMemo(() => React.Children.toArray(children), [children])
+
+    const getConditionalVariant = useCallback(() => {
+      if (childrenArray.some(child => React.isValidElement(child) && child.type === 'em')) {
+        return 'muted'
+      }
+      return 'default'
+    }, [childrenArray])
+
+    const defaultColor = childrenArray.length === 1 ? 'default' : getConditionalVariant()
+
+    return (
+      <Heading
+        ref={ref}
+        className={clsx(defaultColor === 'muted' && styles[`Bento__heading--muted`], className)}
+        size={size}
+        as={as}
+        weight={weight}
+        {...props}
+      >
+        {children}
+      </Heading>
+    )
+  },
+)
+
+type BentoVisualProps = {
+  fillMedia?: boolean
+  position?: string
+  padding?: 'condensed' | 'normal' | 'spacious'
+} & React.HTMLAttributes<HTMLDivElement> &
+  BaseProps<HTMLDivElement>
+
+const Visual = ({fillMedia = true, position = '50% 50%', padding, className, children, ...rest}: BentoVisualProps) => {
+  const childrenToRender = React.Children.map(children, child => {
+    if (React.isValidElement(child)) {
+      if (child.type === 'img') {
+        return React.cloneElement(child as ReturnType<ReactHTML['img']>, {
+          style: {objectPosition: position},
+        })
+      }
+      return child
+    }
+  })
+  return (
+    <div
+      className={clsx(
+        styles.Bento__Visual,
+        !fillMedia && styles['Bento__Visual-no-fill'],
```

Comment:
Could we use the standard double dash for modifiers here? `--`

Reply:
This has been fixed in the following [SHA](https://github.com/primer/brand/pull/390/commits/6b8a1979c1bd4c4c3f8a4bfb1db8b529a2755c2a)